libvast/vast/command.hpp

```diff
@@ -87,7 +84,7 @@ class command {
     return is_root() ? *this : parent_->root();
   }
 
-  inline std::string_view name() const noexcept {
+  std::string_view name() const noexcept {
```

Comment:
Why did you remove `inline` but didn't move the implementation to the `.cpp` file? This results in duplicated symbols. Please make it either inline again for internal linkage or move the implementation to the `.cpp` file. I'd prefer `inline` for small functions like this (3 lines or less is usually a good rule of thumb).